groups/models.py

```diff
@@ -94,3 +109,29 @@ def get_absolute_url(self):
         """Return a permalink that'll scroll to this comment on the page."""
         url = reverse('discussion-thread', kwargs={'pk': self.discussion.pk})
         return url + self.get_pagejump()
+
+    def may_be_deleted(self, user):
```

Comment:
I did have `may_delete`, but it looks weird; you write it as `comment.may_delete(user)` and it reads like "hold on, why can this comment delete this user"?